drivers/mm/mm_drv_intel_adsp.h

```diff
@@ -59,21 +59,33 @@ DEVICE_MMIO_TOPLEVEL_STATIC(tlb_regs, DT_DRV_INST(0));
 #define L2_SRAM_BANK_NUM			(L2_SRAM_SIZE / SRAM_BANK_SIZE)
 #define IS_BIT_SET(value, idx)		((value) & (1 << (idx)))
 
+/* Define amount of regions other than core heaps that virtual memory will be split to
+ * currently includes shared heap and oma region and one regions set to 0 as for table
+ * iterator end value.
+ */
+#define VIRTUAL_REGION_COUNT 3
+
+#define CORE_HEAP_SIZE 0x100000
+#define SHARED_HEAP_SIZE 0x100000
+#define OMA_REGION_SIZE 0x100000
+
 /* size of TLB table */
 #define TLB_SIZE DT_REG_SIZE_BY_IDX(DT_INST(0, intel_adsp_mtl_tlb), 0)
 
 /* references counter to physical pages */
 #define UNUSED_L2_START_ALIGNED ROUND_UP(POINTER_TO_UINT(&unused_l2_sram_start_marker), \
 					 CONFIG_MM_DRV_PAGE_SIZE)
 
-
+/* Attribiutes for memory regions */
+#define MEM_REG_ATTR_CORE_HEAP 1U
+#define MEM_REG_ATTR_SHARED_HEAP 2U
+#define MEM_REG_ATTR_OPPURTUNISTIC_MEMORY 4U
```

Comment:
you use OMA_REGION_SIZE and then MEM_REG_ATTR_OPPURTUNISTIC_MEMORY.
For better clarity, use the same name: OMA or OPPORTUNISTIC.
If using OMA, please add comment to definition what does it state for: Opportunistic Memory Allocation

Reply:
Fixed.